client/task-list/task-item.tsx

```diff
@@ -36,22 +60,32 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 	time,
 	content,
 	expanded = false,
+	level = 'l1',
 } ) => {
 	const className = classnames( 'woocommerce-task-list__item', {
 		'is-complete': completed,
+		'is-l2': level === 'l2' && ! completed,
+		'is-l3': level === 'l3' && ! completed,
```

Comment:
Nitpick: I think a class name like `level-2` would be easier to grok than `is-l2`.

Reply:
Updated in: https://github.com/woocommerce/woocommerce-admin/pull/6916/commits/fd91ddbcc95707aafc281f79e3c26477fb505cd2